minimal-distributed.yaml

```diff
@@ -1,5 +1,8 @@
 version: '2'
 services:
+  etcd:
+    image: quay.io/coreos/etcd:v2.2.5
+    command:  -name etcd0 -advertise-client-urls http://etcd:2379,http://etcd:4001 -listen-client-urls http://0.0.0.0:2379,http://0.0.0.0:4001 -initial-advertise-peer-urls http://etcd:2380 -listen-peer-urls http://0.0.0.0:2380  -initial-cluster etcd0=http://etcd:2380 -initial-cluster-state new
```

Comment:
(Pedantic) Why two spaces before `-name`?  Also, as elsewhere, can we break this line to make it readable?